platforms/cpuinfo.go

```diff
@@ -74,27 +74,40 @@ func getCPUInfo(pattern string) (info string, err error) {
 }
 
 func getCPUVariant() string {
-	variant, err := getCPUInfo("Cpu architecture")
-	if err != nil {
-		log.L.WithError(err).Error("failure getting variant")
-		return ""
-	}
+	var variant string
+	if runtime.GOOS == "windows" {
+		switch runtime.GOARCH {
+		case "arm64":
+			variant = "v8"
+		case "arm":
+			variant = "v7"
+		default:
+			variant = "unknown"
+		}
+	} else {
```

Comment:
one final thing (from me): an early return at the end of the GOOS == windows section means the rest (all prior lines) can remain outside of a large `if` section.

Reply:
Amended